kubernetes/kserve/kserve_wrapper/TorchserveModel.py

```diff
@@ -49,76 +45,9 @@ def __init__(self, name, inference_address, management_address, model_dir):
         self.management_address = management_address
         self.model_dir = model_dir
 
-        logging.info("kfmodel Predict URL set to %s", self.predictor_host)
+        logging.info("Predict URL set to %s", self.predictor_host)
         self.explainer_host = self.predictor_host
-        logging.info("kfmodel Explain URL set to %s", self.explainer_host)
-
-    async def predict(self, request: Dict) -> Dict:
```

Comment:
Does KServe v2 support predict and explain ?

Reply:
No KServe v2 support only predict.